### PR TITLE
Replace listserv signup with direct link to Zoom room.

### LINKED ISSUE #43: Remove reference to outdated listserv.
**Do not disclose security vulnerabilities here, please email us instead: 
az-digital-security@list.arizona.edu**

## Conditions of satisfaction
- [x] Should no longer point people to our listserv
- [x] Should give folks a way to join the Friday Meetings
- [x] Needs to reference a new "arizonadigital" aliased Zoom room with appropriate security settings.

## Follow up
- [ ] Retire the listserv

### PR DESCRIPTION
**Pull request by camikazegreen**
Closes #43

### CODE REVIEW

**Review by tadean — DISMISSED** [review]

**Review by bberndt-uaz — DISMISSED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -105,7 +105,7 @@ <h3 class="card-title text-blue d-inline-block">Quickstart 1.0</h3>
             <h2 class="mt-0">Get Involved</h2>
             <p>A team of web-focused volunteers known as Arizona Digital meets weekly to build and test products like <a href="arizona-bootstrap">Arizona Bootstrap</a> and <a href="https://quickstart.arizona.edu/">UA Quickstart.</a></p>
             <ul>
-              <li>Request an invite to the Friday Arizona Digital meetings by subscribing to the <a href="https://list.arizona.edu/sympa/info/ua-digital">UA Digital listserv</a></li>
+              <li>Drop in to one of the Arizona Digital <a href="https://arizona.zoom.us/j/349115539">Friday Meetings on Zoom</a> every week from 10am to Noon.</li>
```
Comment (trackleft) [inline docs/index.html:108]:
```suggestion
              <li>Drop in to one of the Arizona Digital <a href="https://arizona.zoom.us/j/349115539">friday meetings on Zoom</a> every week from 10am to Noon.</li>
```

Comment (trackleft) [reply docs/index.html:108]: Community zoom room
List meetings 

- Friday meeting
- Wednesday meeting
- Wednesday workshop

**Comment by trackleft** [comment]
Depends on https://github.com/az-digital/az_quickstart/issues/2591

**Review by joeparsons — APPROVED** [review]
Updated with new Zoom room link.

**Review by trackleft — APPROVED** [review]

**Pull request merged**